code/controllers/subsystem/ticker.dm

```diff
@@ -364,13 +404,13 @@ var/round_start_time = 0
 				qdel(player)
 
 
-/datum/controller/gameticker/proc/collect_minds()
+/datum/controller/subsystem/ticker/proc/collect_minds()
 	for(var/mob/living/player in GLOB.player_list)
 		if(player.mind)
-			ticker.minds += player.mind
+			SSticker.minds += player.mind

```

Comment:
why reference the `SSticker`, when this....is the `SSticker`?

Reply:
Why was it referenced like this before?

In all seriousness after you have reviewed this to shreds, I will fix this